test/unit_tests/apps/teachers-digital-platform/js/survey/survey-page-spec.js

```diff
@@ -1,109 +1,63 @@
 import { jest } from '@jest/globals';
 import Cookies from 'js-cookie';
-import surveys from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/tdp-surveys.js';
-import {
-  ChoiceField,
-  scrollToEl,
-  surveyPage,
-  progressBar,
-} from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/survey/survey-page.js';
 import {
   ANSWERS_SESS_KEY,
   RESULT_COOKIE,
   SCORES_UNSET_KEY,
 } from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/survey/config.js';
-import * as modals from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/modals.js';
 import HTML_SNIPPET from '../../html/survey-page.js';
 
 const $ = document.querySelector.bind(document);
 
-xdescribe('The TDP survey page', () => {
+describe('The TDP survey page', () => {
+  const modalInit = jest.fn();
+  const close = jest.fn();
+  let scrollToEl, surveyPage, surveys;
+
+  beforeAll(async () => {
+    jest.unstable_mockModule(
+      '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/modals.js',
+      () => ({
+        init: modalInit,
+        close,
+      })
+    );
+
+    const surveyExports = await import(
+      '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/survey/survey-page.js'
+    );
+    scrollToEl = surveyExports.scrollToEl;
+    surveyPage = surveyExports.surveyPage;
+    surveys = await import(
+      '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/tdp-surveys.js'
+    );
```

Comment:
This one needs the `.default` treatment too.

Reply:
e467290